onnx/defs/math/defs.cc

```diff
@@ -280,8 +386,22 @@ ONNX_OPERATOR_SET_SCHEMA(
     13,
     OpSchema()
         .SetDoc(Ceil_ver13_doc)
-        .Input(0, "X", "Input tensor", "T")
-        .Output(0, "Y", "Output tensor", "T")
+        .Input(0, 
+            "X", 
+            "Input tensor", 
+            "T",
+            OpSchema::Single,
+            true,
+            1,
+            OpSchema::NonDifferentiable)
```

Comment:
It's partial differentiable. I am ok to make it non-differentiable for a more conservative setting. If needed, changing from conservative (non-differentiable) to aggressive (differentiable) won't cost us too much.